libs/cms/src/lib/cms.contentful.service.ts

```diff
@@ -345,6 +345,25 @@ export class CmsContentfulService {
     )
   }
 
+  async getOrganizationByReferenceId(
+    referenceId: string,
+    lang: string,
+  ): Promise<Organization> {
+    const params = {
+      ['content_type']: 'organization',
+      include: 10,
+      'fields.referenceIdentifier[match]': referenceId,
```

Comment:
Just wanted to let you know that the [match] will also match those that have the same prefix (not sure if that will ever be an issue though). I wonder if

`'fields.referenceIdentifier': referenceId`

makes more sense 🤔

Reply:
Ah ok, I'll try it

Reply:
Changed, resolved